runtime/doc/autocmd.txt

```diff
@@ -499,8 +499,8 @@ CursorMoved			After the cursor was moved in Normal or Visual
 				mode or to another window.  Also when the text
 				of the cursor line has been changed, e.g. with
 				"x", "rx" or "p".
-				Not triggered when there is typeahead, when
-				an operator is pending or when moving to
+				Not triggered when there is typeahead, while
+				executing commands in a script file, when
 				another window while remaining at the same
```

Comment:
- wording looks off, should be "when moving to..."
- "executing commands in a script file" is pedantic, can be condensed to "executing a script"


```suggestion
				Not triggered when there is typeahead, while
				executing a script, or when moving to
				another window while remaining at the same
```